builtin/logical/ssh/path_config_ca.go

```diff
@@ -228,6 +228,17 @@ func (b *backend) pathConfigCAUpdate(req *logical.Request, data *framework.Field
 
 		return nil, err
 	}
+
+	if generateSigningKey {
+		response := &logical.Response{
+			Data: map[string]interface{}{
```

Comment:
We already have an endpoint that exposes it in raw form; we should have it in a normal Vault API response too, so this is useful as-is.